Code/MinimalLib/minilib.cpp

```diff
@@ -113,7 +127,16 @@ std::string JSMol::get_svg_with_highlights(const std::string &details) const {
     if (!molval.IsInt()) return ("Atom IDs should be integers");
     atomIds.push_back(static_cast<unsigned int>(molval.GetInt()));
   }
-  return svg_(*d_mol, &atomIds);
+  std::vector<unsigned int> bondIds;
+  if (doc.HasMember("bonds") && !doc["bonds"].IsArray()) {
+    return "JSON doesn't contain 'bonds' field, but it is not an array";
```

Comment:
Why not throw an exception?  This looks like the user has to decide whether it is a valid SVG or not.

Reply:
Right, emscripten doesn't really support them.

Reply:
yep. And, realistically this shouldn't happen anyway